website/docs/r/independent_disk.html.markdown

```diff
@@ -0,0 +1,40 @@
+---
+layout: "vcd"
+page_title: "vCloudDirector: vcd_independent_disk"
+sidebar_current: "docs-vcd-independent-disk"
+description: |-
+  Provides a vCloud Director independent disk resource. This can be used to create and delete independent disks.
+---
+
+# vcd\_inserted\_media
+
+Provides a vCloud Director independent disk resource. This can be used to create and delete independent disks.
+
+Supported in provider *v2.1+*
+
+## Example Usage
+
+```
+resource "vcd_independent_disk" "myNewIndependentDisk" {
+  org             = "my-org"
+  vdc             = "my-vcd"
+  
+  name            = "logDisk"
+  size            = "33"
+  bus_type        = "SCSI"
+  bus_sub_type    = "VirtualSCSI"
+  storage_profile = "external"
+}
+```
```

Comment:
It would be nice to have a snippet example of a VM referencing this disk here too. No need for a full VM spec, only the actual part, something like below (note the below snippet is not formatted):

```
resource "vcd_vapp_vm" "TerraformVAppVMFW" {
  vapp_name     = "TerraformVAppFW"
  name          = "TerraformVAppVMFW"

  # ...

  disk {
	    name = "myDisk"
	    bus_number = 1
	    unit_number = 0
	  }

  depends_on = ["vcd_independent_disk. myNewIndependentDisk"]
}
```

Reply:
Done